### PR TITLE
Added some more fees for exchanges

### PR DESCRIPTION
**Pull request by illera88**

### CODE REVIEW

File: js/kraken.js [hunk 1]
```diff
@@ -57,7 +57,89 @@ module.exports = class kraken extends Exchange {
...
+                    'https://support.kraken.com/hc/en-us/articles/201396777-What-are-the-deposit-fees-',
+                    'https://support.kraken.com/hc/en-us/articles/201893608-What-are-the-withdrawal-fees-',
+                ],
+            },
+            'fees': {
+                'trading': { 
+                    'tierBased': true,
+                    'percentage': true,
+                    'taker': [
+                        [0, 0.26 / 100],
```
Comment (kroitor) [inline js/kraken.js]: sorry, we can't do this like that, without changing the other functions that need those values, the taker and maker should be floats...

File: js/bitstamp.js [hunk 2]
```diff
@@ -77,9 +77,53 @@ module.exports = class bitstamp extends Exchange {
...
+                        [20000, 0.24],
+                        [100000, 0.22],
+                        [400000, 0.20],
+                        [600000, 0.15],
+                        [1000000, 0.14],
+                        [2000000, 0.13],
+                        [4000000, 0.12],
+                        [20000000, 0.11],
+                        [20000001, 0.10],
+                    ]
```
Comment (kroitor) [inline js/bitstamp.js]: missing a `,` here

File: js/bitstamp.js [hunk 3]
```diff
@@ -77,9 +77,53 @@ module.exports = class bitstamp extends Exchange {
                 },
             },
             'fees': {
-                'trading': {
-                    'maker': 0.0025,
-                    'taker': 0.0025,
+                'trading': { // for verified account. Anonymous 0.3 on taker
+                    'tierBased': true, 
+                    'percentage': true,
+                    'taker': [
```
Comment (kroitor) [inline js/bitstamp.js]: can't change these values like that without breaking the other code...

Comment (illera88) [reply js/bitstamp.js]: ideas to have that working?

Comment (kroitor) [reply js/bitstamp.js]: revert them back to floats and store the fee tiers in the `exchange.fees.tiers` ?

File: js/bitfinex.js [hunk 4]
```diff
@@ -124,8 +124,93 @@ module.exports = class bitfinex extends Exchange {
...
+                        'BTG': 0,
+                        'OMG': 0.1,
+                        'DATA': 1,
+                        'QASH': 1,
+                        'ETP': 0.01,
+                        'QTUM': 0.01,
+                        'EDO': 0.5,
+                        'AVT': 0.5,
+                        'USDT': 0,
+
```
Comment (kroitor) [inline js/bitfinex.js]: remove empty line from here

File: js/binance.js [hunk 5]
```diff
@@ -114,12 +114,17 @@ module.exports = class binance extends Exchange {
                     ],
                 },
             },
-            'fees': {
+            'fees': { 
+            // https://support.binance.com/hc/en-us/articles/115000583311
```
Comment (kroitor) [inline js/binance.js]: one more tab for this comment to make it align with python requirements

File: js/binance.js [hunk 6]
```diff
@@ -114,12 +114,17 @@ module.exports = class binance extends Exchange {
...
                 'trading': {
+                    'tierBased': false, // false for tier based
+                    'percentage': true,
                     'taker': 0.001,
                     'maker': 0.001,
                 },
                 'funding': {
+                    'tierBased': false,  // true for tier based/progressive
```
Comment (kroitor) [inline js/binance.js]: remove double space before the two slashes opening the comment

File: js/bitfinex.js [hunk 7]
```diff
@@ -124,8 +124,93 @@ module.exports = class bitfinex extends Exchange {
             },
             'fees': {
                 'trading': {
+                    'tierBased': true, 
+                    'percentage': true,
                     'maker': 0.1 / 100,
                     'taker': 0.2 / 100,
+                    'tiers': {
+                        'taker': [
+                            [0, 0.2 / 100],
```
Comment (kroitor) [inline js/bitfinex.js]: 0.2 / 100 → 0.002

File: js/bitlish.js [hunk 8]
```diff
@@ -31,12 +31,16 @@ module.exports = class bitlish extends Exchange {
                 'secret': false,
             },
             'fees': {
-                'trading': {
+                'trading': { 
                     // for verified account. Anonymous 0.3 on taker
+                    'tierBased': false, 
+                    'percentage': true,
                     'taker': 0.2 / 100,
                     'maker': 0 / 100,
```
Comment (kroitor) [inline js/bitlish.js]: 0 / 100 → 0

File: js/bitmarket.js [hunk 9]
```diff
@@ -102,11 +102,73 @@ module.exports = class bitmarket extends Exchange {
                 'LTC/PLN': { 'id': 'LTCPLN', 'symbol': 'LTC/PLN', 'base': 'LTC', 'quote': 'PLN' },
                 'LTC/BTC': { 'id': 'LTCBTC', 'symbol': 'LTC/BTC', 'base': 'LTC', 'quote': 'BTC' },
                 'LiteMineX/BTC': { 'id': 'LiteMineXBTC', 'symbol': 'LiteMineX/BTC', 'base': 'LiteMineX', 'quote': 'BTC' },
-            },
+            },           
             'fees': {
-                'trading': {
-                    'maker': 0.0015,
-                    'taker': 0.0045,
+                'trading': { // for verified account. Anonymous 0.3 on taker
```
Comment (kroitor) [inline js/bitmarket.js]: move this comment to new line

File: js/bitmarket.js [hunk 10]
```diff
@@ -102,11 +102,73 @@ module.exports = class bitmarket extends Exchange {
...
+                            [49999.99, 0.4 / 100],
+                            [99999.99, 0.3 / 100],
+                            [199999.99, 0.2 / 100],
+                            [299999.99, 0.1 / 100],
+                            [499999.99, 0.0 / 100],
+                        ],
+                    },
+                },
+                'funding': {
+                    'tierBased': false,  // true for tier based/progressive
```
Comment (kroitor) [inline js/bitmarket.js]: same double space before the comment, everywhere – there should be only one (c)

File: js/tidex.js [hunk 11]
```diff
@@ -27,6 +27,43 @@ module.exports = class tidex extends liqui {
...
+                        'ETH': 0.01,
+                        'LTC': 0.001,
+                        'DOGE': 0.01,
+                        'ICN': 2,
+                        'DASH': 0.002,
+                        'GNO': 2,
+                        'EOS': 2,
+                        'BCH': 2,
+                        'USDT': 0,
+
```
Comment (kroitor) [inline js/tidex.js]: this empty line also break it

File: js/tidex.js [hunk 12]
```diff
@@ -27,6 +27,43 @@ module.exports = class tidex extends liqui {
                 'doc': 'https://tidex.com/public-api',
                 'fees': 'https://tidex.com/pairs-spec'
```
Comment (kroitor) [inline js/tidex.js]: most likely need a comma at the end of this line

File: js/kraken.js [hunk 13]
```diff
@@ -57,7 +57,93 @@ module.exports = class kraken extends Exchange {
...
+                        'XMR': 0,
+                        'DASH': 0,
+                        'GNO': 0,
+                        'EOS': 0,
+                        'BCH': 0,
+                        'USD': 5, // if domestic wire
+                        'EUR': 0, // free deposit if EUR SEPA Deposit
+                        'CAD': 5, // if domestic wire
+                        'JPY': 0, //  Domestic Deposit (Free, ¥5,000 deposit minimum)
+
```
Comment (kroitor) [inline js/kraken.js]: this empty line is copy-pasted apparently

**Comment by kroitor** [comment]
Reference: #40

**Pull request merged**